### ISSUE TITLE
Pre-packaged Jira Plugin stopped working `[EROR] unable to start plugin: jira: timed out waiting for plugin`

### ISSUE THREAD

**Issue by towolf** [mattermost/mattermost-server#8399]
#### Summary
We were using the Jira plugin shipped with Mattermost and this stopped working in the most recent release 4.7.1

#### Steps to reproduce
```
Mär 03 11:39:49 s1264 platform[6390]: [2018/03/03 11:39:49 CET] [INFO] API version 3 is scheduled for deprecation. Please see https://api.mattermost.com for details.
Mär 03 11:39:49 s1264 platform[6390]: [2018/03/03 11:39:49 CET] [INFO] Starting 16 websocket hubs
Mär 03 11:39:50 s1264 platform[6390]: open /dev/null: permission denied
Mär 03 11:39:52 s1264 platform[6390]: [2018/03/03 11:39:52 CET] [EROR] unable to start plugin: jira: timed out waiting for plugin
Mär 03 11:39:52 s1264 platform[6390]: [2018/03/03 11:39:52 CET] [INFO] Starting workers
```

#### Observed behavior (that appears unintentional) 

Trying to "Activate" the plugin stalled the UI for a bit, and then it reverted back to Inactive.
The stdout (not the log!) show permission error on `/dev/null`

I think the problem is the `pivot_root` under `/tmp` and something related to `nodev` mount options.

#### Possible fixes

I managed to workaround this by addin these two lines to the systemd mattermost.service file:
```
PrivateTmp=true
PrivateDevices=true
```

**Comment by lindy65** [comment]
Hi @towolf,

Thanks for your report,

Could we have your help upgrading to [v4.7.2](https://about.mattermost.com/download/) to see whether the issue still occurs for you?

Perhaps [this documentation](https://docs.mattermost.com/integrations/jira.html) may also help you to troubleshoot the issue?

**Comment by towolf** [comment]
@lindy65 still occurs on 4.7.2.

**Comment by lindy65** [comment]
Hi @towolf 

Thanks for checking,

I'll ask our engineers for some help troubleshooting your issue 👍

**Comment by jwilander** [comment]
@towolf what OS (+ version) are you running Mattermost on?

In 4.7 we added plugin sandboxing for some linux distros, @ccbrown do you think that could be the cause?

**Comment by ccbrown** [comment]
@jwilander Yeah, it's definitely related to sandboxing. We have another customer with the same issue, and I have a ticket to look into it.

**Comment by ccbrown** [comment]
@towolf Thanks. This report is really helpful. I'll make sure this gets addressed in 4.8.

**Comment by lindy65** [comment]
Thanks @ccbrown - do we have a ticket for this issue that you could post back here for continuity?

**Comment by ccbrown** [comment]
These:

https://mattermost.atlassian.net/browse/MM-9716
https://mattermost.atlassian.net/browse/MM-9742

And this PR will fix the breakage in 4.8 (though the service file changes are probably still a good idea):

https://github.com/mattermost/mattermost-server/pull/8418

**Comment by lindy65** [comment]
Thanks @ccbrown :)

I'll close off this issue here for now and we can track via the tickets you've linked above 👍
